src/test/resources/model/rbac_policy.csv

```diff
@@ -1,5 +1,2 @@
-p, alice, data1, read
-p, bob, data2, write
-p, data2_admin, data2, read
-p, data2_admin, data2, write
-g, alice, data2_admin
\ No newline at end of file
+p, alice, /data1, GET
+p, bob, /data2, POST
```

Comment:
why don't use a classic RBAC policy? https://github.com/jcasbin/jdbc-adapter/blob/master/examples/rbac_policy.csv

read -> GET
write -> POST

Reply:
Because I want to test the currently logged in user have access to the interface.